### PR TITLE
fix/changed No opportunity to load opportunities

### LINKED ISSUE #44: Changing the "No Opportunities " to "Loading Opportunities if any" on all components
This usually creates a dilemma of no opportunities so users generally close the website immediately, we can make this change so that users stay and wait for the results to appear

### PR DESCRIPTION
**Pull request by promaroy**
# Description

changed "No opportunities, sorry" to "loading opportunities if any" while loading a page.

Fixed #44 

## Type of change

Please check options that are relevant to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/Components/ViewOpportunity/index.js [hunk 1]
```diff
@@ -58,7 +59,7 @@ export function ViewOpportunity(props) {
             marginBottom: '200px',
           }}
         >
-          No opportunities, sorry!
+          Loading Opportunities if any...
         </h3>
       </div>
     );
```
Comment (abdus) [inline src/Components/ViewOpportunity/index.js:74]: now, what if there are no opportunities at all? would not the users be waiting forever for some non-existent opportunities to be loaded? (which obviously would not load) 

what you could do is, 
- first, check if the `data` variable is `undefined`. If yes, show the loader to the user. 
- if the `data` variable is not `undefined` and is an array of length 0 `(data && Array.isArray(data) && data.length === 0)`, you can show that `no opportunity found`

**Review by abdus — CHANGES_REQUESTED** [review]

File: src/Components/ViewOpportunity/index.js (hunk 1 above)
Comment (promaroy) [reply src/Components/ViewOpportunity/index.js:74]: I have made the necessary changes

**Review by abdus — APPROVED** [review]

**Comment by abdus** [comment]
Hey, please resolve the conflicts. And I will merge.

**Comment by promaroy** [comment]
resolved the conflicts

**Pull request merged**